pages/automatedtestingcodeceptjs.md

```diff
@@ -5,7 +5,7 @@ See [Documentations on Google Docs](https://docs.google.com/document/d/1FFpCPiTY
 ## Motivation
 While writing tests with the Python `unittest` library is awesome, writing tests in CodeceptJS is super simple and much more intuitive.
 
-## Code
+### Code
```

Comment:
No, I didn't change all h2. `Setting Up a Testing Development Environment` and `Current Challenges` are still h2. I changed them according to the title structures in this [google doc](https://docs.google.com/document/d/1FFpCPiTY3VmFtMHmYe78jOzpHLY_6ulhfdxJd6v7jXg/edit?usp=sharing).